test/commands/test/lib/included-files-parser-test.ts

```diff
@@ -1,7 +1,112 @@
 import { IFileDescriptionJson } from "../../../../src/commands/test/lib/test-manifest-reader";
 import { expect } from "chai";
-import { parseIncludedFiles } from "../../../../src/commands/test/lib/included-files-parser";
+import { copyIncludedFiles, parseIncludedFiles } from "../../../../src/commands/test/lib/included-files-parser";
+import * as xmlUtil from "../../../../src/commands/test/lib/xml-util";
+import * as pfs from "../../../../src/util/misc/promisfied-fs";
 import * as os from "os";
+import * as Sinon from "sinon";
+
+describe("copyIncludedFiles", () => {
+  let sandbox: Sinon.SinonSandbox = null;
+
+  beforeEach(() => {
+    sandbox = Sinon.sandbox.create();
+  });
+  afterEach(() => {
+      sandbox.restore();
+  });
+
+  it("should do nothing if null include is sent", async () => {
+    const inputManifest = { schemaVersion: "1.0.0", cliVersion: "1.0.0",  applicationFile: "",
+    files: ["file1", "file2"], testFramework: { name: "", data: "" } };
+    const expectedManifest = { schemaVersion: "1.0.0", cliVersion: "1.0.0",  applicationFile: "",
+      files: ["file1", "file2"], testFramework: { name: "", data: "" } };
+
+    await copyIncludedFiles(inputManifest, null, null);
+
+    expect(inputManifest).to.deep.equal(expectedManifest);
+  });
+
+  it("should do nothing if empty include is sent", async () => {
+    const inputManifest = { schemaVersion: "1.0.0", cliVersion: "1.0.0",  applicationFile: "",
+    files: ["file1", "file2"], testFramework: { name: "", data: "" } };
+    const expectedManifest = { schemaVersion: "1.0.0", cliVersion: "1.0.0",  applicationFile: "",
+      files: ["file1", "file2"], testFramework: { name: "", data: "" } };
+
+    await copyIncludedFiles(inputManifest, [], null);
+
+    expect(inputManifest).to.deep.equal(expectedManifest);
+  });
+
+  it("should do nothing if something.dll.config exists in include", async () => {
```

Comment:
The is part does not mention that fact that something.dll.config does not contain valid xml.

Not sure what the right behaviour:

1) always exclude *.dll.config files that does not contain valid xml
2) only exclude *.dll.config files if they do not contain valid xml AND a .dll file exists.

Both strategies will avoid the error, but 1) is more restrictive excludes some "valid"/working case 

Seems like you are opting for 1)